ocs_ci/framework/__init__.py

```diff
@@ -14,7 +14,7 @@
 from dataclasses import dataclass, field, fields
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
-DEFAULT_CONFIG_PATH = os.path.join(THIS_DIR, "conf/default_config.yaml")
+DEFAULT_CONFIG_PATH = os.path.join(THIS_DIR, "conf/common_config.yaml")
```

Comment:
We will also need to update any paths to the old `default_config.yaml` in our doc